services/web/server/src/simcore_service_webserver/application_settings.py

```diff
@@ -226,6 +231,19 @@ def log_level(self) -> int:
     def valid_log_level(cls, value) -> str:
         return cls.validate_log_level(value)
 
+    @validator("SC_HEALTHCHECK_TIMEOUT", pre=True)
+    @classmethod
+    def get_healthcheck_timeout_in_seconds(cls, v):
```

Comment:
While I think it's ok to give flexibility, I don't see the benefits here.

I'd rather go for the best unit here, which looks to be milliseconds. At that point I'd rename the env var to `SC_HEALTHCHECK_TIMEOUT_MS` (pattern already used) and add a note in the description.

Reply:
Note that heathchecks we use are of the order of seconds or tens of seconds. I do not see the need to set this in ms

Reply:
You are confusing me. That why even support minutes and milliseconds below since you would like to express everything in seconds?

Reply:
I agree that it would be wise to use the same timekeeing unit (MS) throughout the app. But large integer numbers also are not very clear. Difficult decision.

Reply:
> You are confusing me. That why even support minutes and milliseconds below since you would like to express everything in seconds?

Let's see. Here are two facts that let me take this decision:
1. https://docs.docker.com/engine/reference/builder/#healthcheck options use units
2. this timeout is used in an ``asyncio.wait_for(coro, timeout)`` where the parameter [timeout is in secods](https://docs.python.org/3/library/asyncio-task.html#asyncio.wait_for)